### PR TITLE
Added a tool entry for apitransformer.com

### PR DESCRIPTION
**Pull request by apimatic**
Fixed conflicts and merged. Thanks

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -689,7 +689,21 @@ <h3 class="tooling__title">
             </a>
             <span class="block smaller">"SDK Generator for REST APIs"</span>
         </h3>
-        <p class="tooling__text">Generates SDKs in various programming languages using API descriptions, such as API Blueprint. Also offers a Code-generation-as-a-Service API for CI integration.</code></p>
+        <p class="tooling__text">Generates SDKs in various programming languages using API descriptions, such as API Blueprint. Also offers a Code-generation-as-a-Service API for CI integration.</p>
+    </div></li>
+	
+	<li class="tooling__item"><div class="tooling__item--wrap">
```
Comment (pksunkara) [inline index.html]: You do not have the correct indentation for this line.

Comment (pksunkara) [inline index.html]: Please add this at the last of the list. We follow first come first show.

**Comment by pksunkara** [comment]
Reviewed. Please also remove the merge commits and clean the git history in this PR.

File: index.html (hunk 1 above)
Comment (apimatic) [inline index.html]: Indentation is already messed up in this file. For example see lines 782 and 788, one has 6 spaces, other has 8 spaces. There are several such issues. I don't know what are you expecting to be fixed here. Also, how to "clean" git merge history? Please give clearer instructions. Thanks.

**Comment by apimatic** [comment]
See now. Had to fix many indentation issues.

**Comment by pksunkara** [comment]
You can use git interactive rebase to fix the git history. https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

I can merge this once it is done. Thanks for the PR :smile: :balloon:

**Pull request closed without merging**